src/main/java/seedu/duke/ui/Ui.java

```diff
@@ -57,6 +67,10 @@ public static void printHelpMessage() {
         printLineSeparator();
     }
 
+    /**
+     * Prints error message.
+     * @param e exception whose message we want to print.
+     */
```

Comment:
Hey! Just here to inform you that JavaDoc is not necessary for methods or variables that are self-explanatory 😃